webapp/client/src/stories/FormFieldDate.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import FormFieldDate from "../components/FormFieldDate";
+import FormRowCentered from "../components/FormRowCentered";
+import StepForm from "../components/StepForm";
+import { Default as StepFormDefault } from "./StepForm.stories";
+
+export default {
+  title: "Form Fields/Date",
+  component: FormFieldDate,
+};
+
+const Template = (args) => (
+  <StepForm {...StepFormDefault.args}>
+    <FormRowCentered>
```

Comment:
I'm guessing we don't need the `<FormRowCentered>` here? I can't remember why I added it to the other fields' stories, it's probably not needed there either…